operations/iuf/workflows/admin_directory.md

```diff
@@ -69,6 +69,21 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
          suffix: "-test01"
        ```
 
+       **NOTE:** When installing USS 1.1 or higher, select either SLURM or PBS Pro Products to use on the system before running this stage. For more information, see the `deliver-product` stage
+       details in the "Install and Upgrade Framework" section of the _HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)_.
+       If installing USS 1.1 or higher, the `deploy_slurm` and `deploy_pbs` keys should be in the `site_vars.yaml` file as shown below.
+
+       ```text
+       default:
+        network_type: "cassini"
+        suffix: "-test01"
+        system_name: "my-system"
+        site_domain: "my-site-domain.net"
+       uss:
+        deploy_slurm: true
+        deploy_pbs: true
```

Comment:
Customers will want to install either 0 or 1 WLM, not both. The example should either have false for both of them, or true for only Slurm (it's more popular).

Reply:
fixed, thanks